piptools/scripts/compile.py

```diff
@@ -231,7 +231,7 @@ def cli(
     annotate: bool,
     upgrade: bool,
     upgrade_packages: Tuple[str, ...],
-    output_file: Optional[LazyFile],
+    output_file: Union[LazyFile, Union[IO[Any]], None],
```

Comment:
I don't understand why the internal union has just one arg. This seems pointless.

Reply:
That was undesired, leftover now removed.